examples/asr/transcribe_speech.py

```diff
@@ -187,7 +187,7 @@ class TranscriptionConfig:
     att_context_size: Optional[list] = None
 
     # Use this for model-specific changes before transcription
-    model_change: ModelChangeConfig = ModelChangeConfig()
+    model_change: ModelChangeConfig = field(default_factory=ModelChangeConfig)
```

Comment:
Same as above

Reply:
I get this error if I use the object instead of the class:

---------------------------------------------------------------------
transcribe_speech.py 123 <module>
class TranscriptionConfig:

transcribe_speech.py 190 TranscriptionConfig
model_change: ModelChangeConfig = field(default_factory=ModelChangeConfig())

<string> 3 __init__


TypeError:
'ConformerChangeConfig' object is not callable